src/constants.ts

```diff
@@ -35,3 +35,271 @@ export const ExternalUrls = {
+    { value: 'Seychelles', text: 'Seychelles' },
+    { value: 'Sierra Leone', text: 'Sierra Leone' },
+    { value: 'Singapore', text: 'Singapore' },
+    { value: 'Sint Maarten (Dutch part)', text: 'Sint Maarten (Dutch part)' },
+    { value: 'Slovakia', text: 'Slovakia' },
+    { value: 'Slovenia', text: 'Slovenia' },
+    { value: 'Solomon Islands', text: 'Solomon Islands' },
+    { value: 'Somalia', text: 'Somalia' },
+    { value: 'South Africa', text: 'South Africa' },
+    { value: 'South Georgia and the South Sandwich Islands', text: 'South Georgia and the South Sandwich Islands' },
+    { value: 'South Sudan', text: 'South Sudan' },
+    { value: 'Spain', text: 'Spain' },
+    { value: 'Sri Lanka', text: 'Sri Lanka' },
+    { value: 'Sudan', text: 'Sudan' },
+    { value: 'Suriname', text: 'Suriname' },
+    { value: 'Svalbard and Jan Mayen', text: 'Svalbard and Jan Mayen' },
+    { value: 'Sweden', text: 'Sweden' },
+    { value: 'Switzerland', text: 'Switzerland' },
+    { value: 'Syrian Arab Republic', text: 'Syrian Arab Republic' },
+    { value: 'Taiwan, Province of China', text: 'Taiwan, Province of China' },
+    { value: 'Tajikistan', text: 'Tajikistan' },
+    { value: 'Tanzania, United Republic of', text: 'Tanzania, United Republic of' },
+    { value: 'Thailand', text: 'Thailand' },
+    { value: 'Timor-Leste', text: 'Timor-Leste' },
+    { value: 'Togo', text: 'Togo' },
+    { value: 'Tokelau', text: 'Tokelau' },
+    { value: 'Tonga', text: 'Tonga' },
+    { value: 'Trinidad and Tobago', text: 'Trinidad and Tobago' },
+    { value: 'Tunisia', text: 'Tunisia' },
+    { value: 'Turkey', text: 'Turkey' },
+    { value: 'Turkmenistan', text: 'Turkmenistan' },
+    { value: 'Turks and Caicos Islands', text: 'Turks and Caicos Islands' },
+    { value: 'Tuvalu', text: 'Tuvalu' },
+    { value: 'Uganda', text: 'Uganda' },
+    { value: 'Ukraine', text: 'Ukraine' },
+    { value: 'United Arab Emirates', text: 'United Arab Emirates' },
+    { value: 'United States Minor Outlying Islands', text: 'United States Minor Outlying Islands' },
+    { value: 'United States', text: 'United States' },
+    { value: 'Uruguay', text: 'Uruguay' },
+    { value: 'Uzbekistan', text: 'Uzbekistan' },
+    { value: 'Vanuatu', text: 'Vanuatu' },
+    { value: 'Venezuela, Bolivarian Republic of', text: 'Venezuela, Bolivarian Republic of' },
+    { value: 'Viet Nam', text: 'Viet Nam' },
+    { value: 'Virgin Islands, British', text: 'Virgin Islands, British' },
+    { value: 'Virgin Islands, U.S.', text: 'Virgin Islands, U.S.' },
+    { value: 'Wallis and Futuna', text: 'Wallis and Futuna' },
+    { value: 'Western Sahara', text: 'Western Sahara' },
+    { value: 'Yemen', text: 'Yemen' },
+    { value: 'Zambia', text: 'Zambia' },
+    { value: 'Zimbabwe', text: 'Zimbabwe' },
+] as const;
+
+const defaultAddress = {
+    premises: '',
+    postCode: '',
+    country: '',
+    addressLine1: '',
+    townOrCity: ''
+} as const;
```

Comment:
There is no point having this as const.
It should aslo have the correct type which would be the `Address` type from the pirvate sdk.
```suggestion
const defaultAddress = {
    premises: '',
    postCode: '',
    country: '',
    addressLine1: '',
    townOrCity: ''
} as Address;
```